core/controllers/base_test.py

```diff
@@ -1202,6 +1247,13 @@ def test_no_error_is_raised_on_opening_new_tab_after_signup(self):
 
         self.get_html_response('/community-library')
 
+    def test_500_error_is_raised_when_enable_user_creation_is_false(self):
+        self.login('abc@example.com')
+
+        with self.swap(feconf, 'ENABLE_USER_CREATION', False):
+            response = self.get_response_without_checking_for_errors(
+                feconf.SIGNUP_URL + '?return_url=/', [500])
```

Comment:
Nit, but only address if there are other review comments: prefer string interpolation to concatenation.

Reply:
Done